SourceCode/frontend/src/Modules/Events/EventContainer.js

```diff
@@ -0,0 +1,114 @@
+import React, { useState } from 'react';
+import FooterContainer from '../Footer/FooterContainer';
+import EventHeaderComponent from './EventHeaderComponent';
+import SearchComponent from '../General/SearchComponent';
+import EventListComponent from './EventListComponent';
+import '../../Assets/scss/main.scss';
+import ResourceListComponent from './ResourceListComponent';
+import { ProgressBar } from 'react-bootstrap';
+import {API_URL} from '../../Utils/Urls';
+import { EventDateSorter, EventHandler } from '../../Utils/EventHandler';
+import axios from 'axios';
+
+const EventContainer = props => {
+    const [foodBankResponse, setFoodBankResponse] = useState(false);
+    const [agencyResponse, setAgencyResponse] = useState(false);
+    let [foodBankData,setFoodBankData] = useState({});
+    let [agencyData,setAgencyData] = useState({});
+    let [searchDetails,setSearchDetails] = useState({});
+    const [serverError, setServerError] = useState(false);
+    const [loading, setLoading] = useState(false);
+    let isSearchData = !!props.location.state;
+
+    React.useEffect(() => {
+        if (isSearchData){
+            buildSearchData(props.location.state.searchData);
+            props.history.replace({ state: null });
+        }
+    }, []);
+
+    const handleSubmit = async query => {
+        if(query) {
+            setLoading(true);
+            let foodBankUri = API_URL.FOODBANK_LIST;
+            const { zip_code } = query;
+            // Going to use axios for now
+            try {
+                const resp = await axios.get(foodBankUri, { params: { zip_code } });
+                const { data } = resp;
+                setFoodBankData(data);
+                setFoodBankResponse(true);
+                setLoading(false);
+            } catch (err) {
+                setServerError(true);
+                setLoading(false);
+            }
+
+            // This along with EventList should be in it's own container.
```

Comment:
Since this is a separate API this should be in it's own Container along with all the other Event List stuff. 
Just ran out of time this weekend to do that.